### PR TITLE
Enrich uncountable dictionary

### PR DESCRIPTION
**Pull request by PavelTkachenko**
1. Add frequently used words
2. Move words to separate file

### CODE REVIEW

**Comment by GustavoCaso** [comment]
@PavelTkachenko maybe we could have a separate text file with all the words, that get loaded by `Dictionary::Uncountable`.

Thanks for the PR ❤️

File: lib/dry/inflector/inflections/dictionary/uncountable.rb [hunk 1]
```diff
@@ -0,0 +1,121 @@
+# frozen_string_literal: true
+
+module Dry
+  class Inflector
+    class Inflections
+      module Dictionary
+        module Uncountable
+          def self.list
```
Comment (solnic) [inline lib/dry/inflector/inflections/dictionary/uncountable.rb]: WDYT about simply setting `Uncountable` as a constant instead of having a method?

Comment (GustavoCaso) [reply lib/dry/inflector/inflections/dictionary/uncountable.rb]: That could work as well 😄. I mention the option for the file based on the idea that would be easier to maintain in the long run

Comment (PavelTkachenko) [reply lib/dry/inflector/inflections/dictionary/uncountable.rb]: Thanks for reply!  I will make it.

**Comment by GustavoCaso** [comment]
@PavelTkachenko any update regarding the comments?

**Comment by AMHOL** [comment]
Where is this list from? Some of them don't seem right, for example `beer`.

**Comment by flash-gordon** [comment]
I will close this PR. AS has a short list of uncountable words, I don't see why we wouldn't do the same. Our current list includes all the AS exceptions plus adds a few more. If you want to modify it, it's supported via `irregular`. Besides all this, the suggested list is simply not correct :)

**Pull request closed without merging**